sdks/java/io/solr/src/main/java/org/apache/beam/sdk/io/solr/SolrIO.java

```diff
@@ -464,9 +460,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
     }
 
     @Override
-    public void validate() {
-      spec.validate(null);
-    }
+    public void validate() {}
```

Comment:
Done (had to introduce a default empty validate() method in Source, which is probably a good thing anyway, and I removed a bunch more empty validate() methods)